projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts

```diff
@@ -35,6 +35,7 @@ describe('ng-update', () => {
 import { TuiButtonOptions, some } from '@taiga-ui/core';
 
 const options: TuiButtonOptions = {};
+const config  = {};
```

Comment:
```suggestion
const config = {};
```

Reply:
I think
```ts
const config: WithDateMaskPipeConfig = {}
```
should become
```ts
const config: any = {};
```

Why:
Imagine user has
```ts
const config: TuiBreadCrumbsItem = {caption: '', routerLink: ''};
// ...

console.log(config.routerLinkOptions);
```
after migration:
```ts
const config = {caption: '', routerLink: ''};
// ...

// TS2339: Property 'routerLinkOptions' does not exist on type ...
console.log(config.routerLinkOptions);
```

Reply:
Nice idea, thanks:)